app/controllers/bridges/lom/tasks_controller.rb

```diff
@@ -0,0 +1,208 @@
+          schema = Nokogiri::XML::Schema(File.read(OML_SCHEMA_PATH))
+          errors = schema.validate(builder.doc)
+
+          return render plain: errors.map(&:message).join("\n") if errors.any?
+        end
+
+        render xml: builder
+      end
+
+      private
+
+      def sample_oml(xml)
+        xml.lom(xmlns: 'http://ltsc.ieee.org/xsd/LOM') do
+          oml_general(xml)
+          oml_lifecycle(xml)
+          oml_meta_metadata(xml)
+          oml_technical(xml)
+          oml_educational(xml)
+          oml_rights(xml)
+          # Other top-level elements: relation, annotation, and classification.
+        end
+      end
+
+      # rubocop:disable Metrics/AbcSize
+      # rubocop:disable Metrics/BlockLength
+      # rubocop:disable Metrics/MethodLength
+      def oml_general(xml)
+        xml.general do
+          xml.identifier do
+            xml.catalog 'UUID'
+            xml.entry task.uuid
+          end
+          xml.title do
+            xml.string task.title, language: task_lang
+          end
+          xml.language task_lang
+          xml.description do
+            xml.string task.description, language: task_lang
+          end
+          if task.programming_language&.language.present?
+            xml.keyword do
+              xml.string "programming language: #{task.programming_language.language}", language: 'en'
+            end
+            if task.programming_language&.version.present?
+              xml.keyword do
+                xml.string "programming language version: #{task.programming_language.version}", language: 'en'
+              end
+            end
+          end
+          xml.structure do
+            xml.value 'atomic'
+          end
+          xml.aggregationLevel do
+            xml.value 1
+          end
+        end
+      end
+      # rubocop:enable all
+
+      def oml_lifecycle(xml)
+        xml.lifeCycle do
+          xml.version do
+            xml.string task_version, language: 'en'
+          end
+          xml.status do
+            xml.value 'final'
+          end
+          xml.contribute do
+            xml.role do
+              xml.value 'author'
+            end
+            xml.entity task_author_vcard
+            xml.date do
+              # TODO: We omit the time part, since the regex provided by the xsd schema seems to be broken regarding
+              # the time zone part. Try a validated request, e.g., with '1997-07-16T19:20:30+01:00'. It fails, but
+              # it is an example from the IEEE Standard document.
+              xml.dateTime task.updated_at.to_date.iso8601
+            end
+          end
+        end
+      end
+
+      def oml_meta_metadata(xml)
+        xml.metaMetadata do
+          xml.metadataSchema 'ProFormA MD 1.0'
+        end
+      end
+
+      def oml_technical(xml)
+        xml.technical do
+          xml.format 'text/xml'
+          xml.location task_url(id: 'sample')
+          xml.location download_task_url(id: 'sample')
```

Comment:
I am fine with this even though it somehow feels incorrect to add the `task_url` without specifying the format. For now, I would keep it this way (because both locations should be included) and postpone this to a later stage.